src/app/portlets/content-types/fields/content-type-fields-drop-zone/content-type-fields-drop-zone.component.html

```diff
@@ -19,7 +19,7 @@
     </div>
 </div>
 <p-dialog 
-    [header]="formData && formData.id ? i18nMessages['edit'] : i18nMessages['created']"
+    [header]="formData && formData.id ? i18nMessages['contenttypes.dropzone.action.edit'] : i18nMessages['contenttypes.dropzone.action.create.field']"
```

Comment:
'contenttypes.dropzone.action.create.field' is not in the languages.properties

maybe?

contenttypes.fields.dropzone.action.edit

Reply:
Right, 'contenttypes.dropzone.action.create.field' was missing in the language.properties, it was added in the latest commit.